### ISSUE TITLE
more diagnostic plots

### ISSUE THREAD

**Issue by nscherf** [neural-data-science-lab/intro_dgm#2]
For the VAE model we visualise generated samples per epoch and the validation loss curve. It would be good to have additional diagnostic plots:

- [ ] side-by-side comparison of input and reconstruction for a few samples
- [ ] for MSE / Gaussian loss function: plot of mean and variance estimate for a few samples
